### PR TITLE
Initial changes into switchblade-next

### PR DESCRIPTION
**Pull request by Doges**
These are the initial changes to the Switchblade@next

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,25 +22,25 @@
...
-    "eslint": "^7.32.0",
+    "@types/signale": "^1.4.4",
+    "@typescript-eslint/eslint-plugin": "^5.39.0",
+    "@typescript-eslint/parser": "^5.39.0",
+    "eslint": "^8.24.0",
     "eslint-config-google": "^0.14.0",
-    "tsc-watch": "^4.4.0",
-    "typescript": "^4.3.5"
+    "tsc-watch": "^5.0.3",
+    "typescript": "^4.8.4"
```
Comment (almeidx) [inline package.json:44]: Should use the ~ modifier here as typescript doesn't follow semver

Comment (Doges) [reply package.json:44]: verdade, as vezes esqueço que trocou pra ts

**Review by almeidx — CHANGES_REQUESTED** [review]
Why is there a yarn.lock and a package-lock.json?

**Comment by Doges** [comment]
> Why is there a yarn.lock and a package-lock.json?

@jpsl00 added it

**Comment by almeidx** [comment]
You shouldn't mix package managers. Pick one

**Comment by Doges** [comment]
> 

calm down it was already in there

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM node:16
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
+COPY . .
+CMD [ "npm", "start" ]
```
Comment (almeidx) [inline Dockerfile:6]: Missing a build step

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FROM node:16
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
```
Comment (almeidx) [inline Dockerfile:4]:
```suggestion
RUN npm ci
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,7 @@
+FROM node:16
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm ci
+COPY . .
+CMD [ "npm", "run", "build" ]
```
Comment (almeidx) [inline Dockerfile:6]: AFAIK you can only have 1 CMD? Build should be a regular RUN

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
```
Comment (almeidx) [inline Dockerfile:6]:
```suggestion
RUN npm run build
CMD [ "npm", "start" ]
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by almeidx — APPROVED** [review]

**Pull request merged**